src/components/calcite-pick-list-item/calcite-pick-list-item.e2e.ts

```diff
@@ -26,4 +26,29 @@ describe("calcite-pick-list-item", () => {
     await item.click();
     expect(await item.getProperty("selected")).toBe(false);
   });
+
+  it("should fire event calciteListItemChange when item is clicked", async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(`<calcite-pick-list-item text-label="test" value="example"></calcite-pick-list-item>`);
+    const item = await page.find("calcite-pick-list-item");
+    await page.evaluate(() => {
+      document.addEventListener("calciteListItemChange", (event: CustomEvent) => {
+        (window as any).eventDetail = event.detail;
+      });
+    });
+
+    await item.click();
+
+    await page.waitForChanges();
+
+    const eventDetail: any = await page.evaluateHandle(() => {
```

Comment:
Can you type the detail object? This type could also be used in event handlers:

```tsx
// assuming ItemChangeDetail is defined somewhere
calciteListItemChangeHandler(
  this: CalcitePickList | CalciteValueList, 
  event: CustomEvent<ItemChangeDetail>
): void {
```

Aside: I just noticed that we can also type the EventEmitters, that could be useful too.